### PR TITLE
Raoul/eol

### PR DESCRIPTION
**Pull request by raoulj**
[Ticket](https://mail.google.com/mail/u/0/#box/agxzfm1haWxmb29nYWVyMAsSDE9yZ2FuaXphdGlvbiIRb2lzbWFpbEBnbWFpbC5jb20MCxIEQ2FzZRiho4JuDA)

### CODE REVIEW

File: resources/bootstrap.js [hunk 1]
```diff
@@ -6,11 +6,91 @@
...
+
+const makelink = function(text, url) {
+  var a = document.createElement("a");
+  a.appendChild(document.createTextNode(text));
+  a.title = text;
+  a.href = url;
+  return a;
+}
+
+const hasModelBeenClosedKey = "StreakSecureGmail.hasClosedDeprecatedModal";
```
Comment (raoulj) [inline resources/bootstrap.js]: Make all caps for global var

File: resources/bootstrap.js [hunk 2]
```diff
@@ -6,11 +6,91 @@
...
+  const content3 = document.createTextNode(" the Secure Gmail extension.\n\nLooking forward, the Streak team will continue to focus on improving our core product - the ");
+  div.appendChild(content3);
+  const link3 = makelink("Streak CRM for Gmail", "http://www.streak.com");
+  div.appendChild(link3);
+  const content4 = document.createTextNode(". It offers workflow management but also power tools like email tracking, scheduled emails, and mail merge all from within Gmail.");
+  div.appendChild(content4);
+
+  const modalView = sdk.Widgets.showModalView({
+    el: div,
+    chrome: true,
```
Comment (raoulj) [inline resources/bootstrap.js]: drop line bc chrome is default

File: resources/bootstrap.js [hunk 3]
```diff
@@ -6,11 +6,91 @@
...
+        type: "PRIMARY_ACTION",
+        color: "blue",
+        text: "Try Streak for Free",
+        onClick: function(e) {
+          track("deprecationModal.linkClicked");
+          window.open(
+            "http://www.streak.com?utm_medium=gmail&utm_source=inboxsdk&utm_campaign=inappmodal"
+          );
+        },
+        orderHint: 5
```
Comment (raoulj) [inline resources/bootstrap.js]: drop orderhints

File: resources/bootstrap.js [hunk 4]
```diff
@@ -6,11 +6,91 @@
...
+          sessionStorage.setItem(hasModelBeenClosedKey, true);
+          e.modalView.close();
+        },
+        orderHint: 5
+      }
+    ]
+  });
+
+  modalView.on('destroy', () => {
+    sessionStorage.setItem(hasModelBeenClosedKey, true);
```
Comment (raoulj) [inline resources/bootstrap.js]: If close button is selected, may be different behavior than "X" button

File: resources/bootstrap.js [hunk 5]
```diff
@@ -6,11 +6,87 @@
...
+  div.appendChild(content1)
+  const link1 = makelink("Confidential Mode", "https://support.google.com/mail/answer/7674059");
+  div.appendChild(link1);
+  const content2 = document.createTextNode(", there’s been less of a need for our extension.  We highly encourage you to transition to this new Gmail feature as soon as possible to ensure you can continue sending confidential emails. You should also ");
+  div.appendChild(content2);
+  const link2 = makelink("uninstall", "https://support.google.com/chrome_webstore/answer/2664769");
+  div.appendChild(link2);
+  const content3 = document.createTextNode(" the Secure Gmail extension.\n\nLooking forward, the Streak team will continue to focus on improving our core product - the ");
+  div.appendChild(content3);
+  const link3 = makelink("Streak CRM for Gmail", "http://www.streak.com");
```
Comment (frew) [inline resources/bootstrap.js]: Make link https?

Comment (raoulj) [reply resources/bootstrap.js]: Ah.  Missed that

**Pull request merged**